Documentation/MixedRealityConfigurationGuide.md

```diff
@@ -1,14 +1,14 @@
-# Mixed Reality Toolkit configuration guide
+# Mixed Reality Toolkit Profile configuration guide
 
 ![](../Documentation/Images/MRTK_Logo_Rev.png)
 
 The Mixed Reality Toolkit centralizes as much of the configuration required to manage the toolkit as possible (except for true runtime "things").
 
-**This guide is a simple walkthrough for each of the configuration screens currently available for the toolkit, more in-depth guides for each of the features is coming soon.**
+**This guide is a simple walkthrough for each of the configuration profile screens currently available for the toolkit, more in-depth guides for each of the features is coming soon.**
 
 Configuration profiles provide reusable blocks of configuration that can be used and swapped out at runtime (with the exception of the InputActions profile) to meet the demands for most Mixed Reality projects. This allows you to style your configuration for different input types (driving vs flying) or different behavior's your project needs.
```

Comment:
I would also remove the point here that can be swapped out at runtime. See Will's comment in PR #4991 . That is the goal and desire but atm we are not there yet. Services aren't reliable enough for teardown or refresh dynamically

Reply:
Removed the part. Thanks!